templates/github/.github/workflows/scripts/stage-changelog-for-default-branch.py.j2

```diff
@@ -61,9 +61,10 @@ except GitCommandError:
         git.cherry_pick("--continue")
 git.reset("origin/{{ plugin_default_branch }}")
 
-# Do not remove changelog entries
-msg = repo.commit().message
-git.reset("HEAD~1")
-git.add("CHANGES.rst")
-git.commit("-m", msg)
-git.stash()
+if not release_version_arg.endswith(".0"):
```

Comment:
So this will make the changelog snippets from minor releases (3.17.0) disappear from main, but not the backported fix from e.g. 3.17.1?

Reply:
yes